src/Service.Host/client/src/components/reports/EanCodes.js

```diff
@@ -1,37 +1,43 @@
 ﻿import React, { Component } from 'react';
-import { Loading } from '../common/Loading';
-import { Grid, Row, Col } from 'react-bootstrap';
-import Table from '../common/Table';
-import ExportButton from '../common/ExportButton';
+import { Loading } from '@linn-it/linn-form-components-library';
+import Grid from '@material-ui/core/Grid';
+import { ReportTable, ExportButton } from '@linn-it/linn-form-components-library';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+    root: {
+      flexGrow: 1,
```

Comment:
Are these flex grows needed (or having an effect) as applied here to the container rather than the grid items?